### PR TITLE
Merge pull request #9 from ibm-hyperknowledge/dev

### PR DESCRIPTION
**Pull request by eltonfss**
- allowing updating connector className 
- Removing useless property.

### CODE REVIEW

File: hkgraph.js [hunk 1]
```diff
@@ -80,29 +81,35 @@ HKGraph.prototype.setEntity = function(entity)
 	// Update parent
 
 	// Clean old entity
-	let oldParent = this.getEntity(oldEntity.parent);
-	if(oldParent)
+	if(oldEntity.parent)
```
Comment (vitornl) [inline hkgraph.js:84]: Can we ensure that ```oldEntity``` **always** has the ```parent``` property? Without ensuring this, this change may causes property not found-like errors. The old-way, i.e., `let oldParent = this.getEntity(oldEntity.parent); if(oldParent)` is able to check whenever ```oldEntity``` indeed has the ```parent``` property.

File: hkgraph.js [hunk 2]
```diff
@@ -80,29 +81,35 @@ HKGraph.prototype.setEntity = function(entity)
...
+		else
+		{
+			delete this.orphans[oldEntity.parent][oldEntity.id];
+		}
 	}
 	
 	// Set new parent
-	let parent = this.getEntity(entity.parent);
-	if(parent || entity.parent === null)
+	if(entity.parent)
```
Comment (vitornl) [inline hkgraph.js:98]: Similar to the last comment.

File: hkgraph.js [hunk 3]
```diff
@@ -68,7 +68,8 @@ HKGraph.prototype.setEntity = function(entity)
 
     if(entity.type === Types.CONNECTOR)
     {
-        oldEntity.roles = entity.roles;
+		oldEntity.roles = entity.roles;
+		oldEntity.className = entity.className
```
Comment (vitornl) [inline hkgraph.js:72]: Missing ```,``` at the end of the line. Not critical though.

File: hkgraph.js (hunk 1 above)
Comment (eltonfss) [reply hkgraph.js:84]: Thanks @vitornl. I will include a verification using **hasOwnProperty**, although in my recent tests I've noticed that JS is able to check the value of an inexistent property without errors. Here follows an example:
image

**Pull request merged**